components/BudgetTimelineForm.tsx

```diff
@@ -165,10 +167,10 @@ const BudgetTimelineForm = ({
                             height: 2
                         }}
                         handleStyle={{
-                            height: 28,
-                            width: 28,
-                            marginLeft: -14,
-                            marginTop: -14,
+                            height: size.screen == 'xs' ? 18 : 28,
+                            width: size.screen == 'xs' ? 18 : 28,
+                            marginLeft: size.screen == 'xs' ? -5 : -14,
+                            marginTop: size.screen == 'xs' ? -8 : -14,
```

Comment:
are these breakpoints the same that we use in our styling system?

Reply:
No, I checked the styling files and didn't see any breakpoints there, I had to play with the sizes to see which one was the best match.